src/Option_Alpha/web/app.py

```
@@ -0,0 +1,155 @@
+    finally:
+        await db.close()
+
+
+# ---------------------------------------------------------------------------
+# App Factory
+# ---------------------------------------------------------------------------
+
+
+def create_app() -> FastAPI:
+    """Create and configure the FastAPI application."""
+    global templates  # noqa: PLW0603
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `PLW0603`)

Remove unused `noqa` directive

(RUF100)